Userland/Applets/Keymap/main.cpp

```diff
@@ -22,7 +22,9 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     window->set_title("Keymap");
     window->resize(16, 16);
     window->show();
-    window->make_window_manager(WindowServer::WMEventMask::KeymapChanged);
+
+    GUI::ConnectionToWindowManagerServer::the().async_set_event_mask(WindowServer::WMEventMask::KeymapChanged);
```

Comment:
This line gets reformatted in the next commit (without any changes as far as I can tell), same with some of the other occurrences of `async_set_event_mask`.

Please run through a rebase again, and make sure to run the formatter and commit the changes that it makes after this commit and after the following one. This ensures that the formatting stays consistent.

Reply:
actually it seemed like clang-format did not do anything to that line, it would accept both